hud_src/resource/ui/hudmatchstatus.res

```diff
@@ -252,47 +240,23 @@
 		"NegativeColor"			"255 0 0 255"
 		"delta_lifetime"		"1.5"
 		"delta_item_font"		"HudFontMediumSmallBold"
-
-		if_match
-		{
-			"xpos"					"cs-0.5"
-			"wide"					"130"
-			"ypos_minmode"			"0"
-			"delta_item_x"			"35"
-			"delta_item_start_y"	"12"
-			"delta_item_end_y"		"50"
-			"PositiveColor"			"0 255 0 255"
-			"NegativeColor"			"255 0 0 255"
-			"delta_lifetime"		"1.5"
-			"delta_item_font"		"HudFontMediumSmallBold"
-		}
 		
-		"TimePanelValue"
-		{
-			"ControlName"		"CExLabel"
-			"fieldName"		"TimePanelValue"
-			"font"			"HudFontTimer"
-			"fgcolor"		"TanLight"
-			"xpos"			"30"
-			"ypos"			"9"
-			"zpos"			"3"
-			"wide"			"50"
-			"tall"			"31"
-			"visible"		"1"
-			"enabled"		"1"
-			"textAlignment"		"center"
-			"labelText"		"0:00"
-
-			if_match
-			{
-				"proportionaltoparent"	"1"
-
-				"xpos"			"cs-0.5"
-				"ypos"			"12"
-				"tall"			"10"
-				"font"			"HudFontTimer"
-			}
-		}	
+	"TimePanelValue"
+	{
+		"ControlName"		"CExLabel"
+		"fieldName"		"TimePanelValue"
+		"font"			"HudFontTimer" //"HudFontMediumSmallBold"
+		"fgcolor"		"White"
+		"xpos"			"32"
+		"ypos"			"11"
+		"zpos"			"3"
+		"wide"			"45"
+		"tall"			"31"
+		"visible"		"1"
+		"enabled"		"1"
+		"textAlignment"		"center"
+		"labelText"		"0:00"
+	}	
```

Comment:
nit: This looks like it's indented improperly (is it a child of ObjectiveStatusTimePanel?)

Reply:
Here's what it looks like
![20210715121731_1](https://user-images.githubusercontent.com/50184129/125772448-e015999f-95c2-4e2e-b19d-5568d80edf73.jpg)

Since meet your match, HudObjectiveStatus is useless, the timer can only be modified in hudmatchstatus, i took the timer from HudObjectiveStatus  and put it in hudmatchstatus

Reply:
Ah I meant strictly in the .res file's formatting itself (tabs), not in the HUD. Actual hud placement seemed fine for this.

Reply:
Oh, you just converted the whole thing to spaces. That works too (I was only talking about the indentation for TimePanelValue but I wasn't too clear saying that).

Reply:
I didn't realize I could do this on PRs, but I've made the relevant change and reverted the tab->space commit.